src/runner.ts

```diff
@@ -94,6 +94,16 @@ export interface Options {
      */
     rulesDirectory?: string | string[];
 
+    /**
+     * Check files via stdin.
+     */
+    stdin?: boolean;
+
+    /**
+     * Tell TSLint where the file for some stdin data is.
```

Comment:
this language is awkward. what is "some stdin data?" please clarify this before merging.

Reply:
Throw whatever string you want at me, and I'll write that.

Reply:
Okay, done. :+1: